crates/router/src/connector/stripe.rs

```diff
@@ -130,12 +130,14 @@ impl
         &self,
         req: &types::PaymentsPreProcessingRouterData,
     ) -> CustomResult<Option<types::RequestBody>, errors::ConnectorError> {
-        let req = stripe::StripeAchSourceRequest::try_from(req)?;
+        let req = stripe::StripeCreditTransferSourceRequest::try_from(req)?;
+        router_env::logger::info!(connector_request=?req);
```

Comment:
Is logging of the connector request intentional?

Reply:
No. I will remove it